tst/EnergyPlus/unit/DXCoils.unit.cc

```diff
@@ -3963,5 +3972,13 @@ TEST_F(EnergyPlusFixture, DXCoils_GetDXCoilCapFTCurveIndexTest)
     EXPECT_DOUBLE_EQ(8763.2701224550547, NominalHeatingDesignCapacity_lowestSpeed);
     EXPECT_DOUBLE_EQ(8945.5439208717980, NominalHeatingDesignCapacity_designSpeed);
 }
+TEST_F(EnergyPlusFixture, DXCoils_RatedInletAirWTest)
+{
+
+    Real64 Tdb = 26.6667;
+    Real64 Twet = 19.4444;
+    Real64 RatedW = Psychrometrics::PsyWFnTdbTwbPb(Tdb, Twet, 101325.0);
+    EXPECT_NEAR(RatedInletAirHumRat, RatedW, 0.000001);
+}
```

Comment:
This is kind of a funny little unit test.  Where is RatedInletAirHumRat declared?

Reply:
@Myoldmopar The RatedInletAirHumRat is defined at line 159 in DXCoils as below:
    
Real64 const RatedInletAirHumRat(0.0111847);      // Humidity ratio corresponding to 80F dry bulb/67F wet bulb

Reply:
I see.  OK, thanks!